lib/util/splitPath.js

```diff
@@ -1,7 +1,8 @@
 import reject from 'lodash/reject';
+import toString from 'lodash/toString';
 
 export default function splitPath(path) {
   return Array.isArray(path) ?
     path :
-    reject(path.split('.'), x => !x);
+    reject(toString(path).split('.'), x => !x);
```

Comment:
I don't think we need all of the things that `lodash/toString` does, can you just use `(path + '')` instead please?